plugins/com.google.cloud.tools.eclipse.appengine.login/.classpath

```diff
@@ -3,5 +3,6 @@
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src/"/>
+	<classpathentry kind="lib" path="lib/google-oauth-client-1.22.0.jar"/>
```

Comment:
Correct. The jar doesn't have the OSGi metadata. Thanks for filing an issue against the jar repo.

I will need to add more jars as we move on, so I will file similar issues to generate OSGi metadata along the way.